pkg/parsers/zero/zero.go

```diff
@@ -0,0 +1,67 @@
+package zero
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type ZeroProgram struct {
+	Attributes []string          `json:"attributes"`
+	Builtins   map[string]int64  `json:"builtins"`
```

Comment:
Could you use the Builtin type we defined for parsers/starknet. This way validation already occurs during parsing

Reply:
I think they are not the same type. The starknet-parser one is uint8.

Reply:
Yes, but it has it's own `Builtin` type, with its implementation of a marsharller/unmarshaller. The same can be reused here if I am not mistaken.